File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1864.cs
using System;

class URI
{
    static void Main() {
        
        while(true) {
            string linha = Console.ReadLine();
            if (string.IsNullOrEmpty(linha))
                break;
            string[] ls = linha.Split(' ');

            int n = int.Parse(ls[0]);
            string frase = "LIFE IS NOT A PROBLEM TO BE SOLVED";
            int i;
            for(i = 0; i < n; i++){
                Console.Write("{0}", frase[i]);
            }
            Console.Write("\n");
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1864.java
import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        int n;
        n = ler.nextInt();
        String f = "LIFE IS NOT A PROBLEM TO BE SOLVED";
        char[] frase = f.toCharArray(); 
        
        for(int i=0;i<n;i++){
            System.out.printf("%c", frase[i]);
        }
        System.out.printf("\n");
    }
}